pkg/ddl/000038_oidc_user.down.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+DROP INDEX email_unique_idx;
+ALTER TABLE auth_users
```

Comment:
Doesn't seem like this drops what it should.  In general, I am not sure we should drop *anything*; instead just make "migrate up" idempotent, then you can migrate down losing **nothing** and migrate back up when you're ready.